plasmapy_nei/eigen/eigenclass.py

```diff
@@ -0,0 +1,354 @@
+"""
+Contains the class to provide access to eigenvalue data for the
+ionization and recombination rates.
+"""
+
+__all__ = ["EigenData"]
+
+import h5py
+import numpy as np
+from numpy import linalg as LA
+import pkg_resources
+import warnings
+
+try:
+    from plasmapy.particles import Particle, particle_input
+except (ImportError, ModuleNotFoundError):
+    from plasmapy.atomic import Particle, particle_input
```

Comment:
I just created #23 and #24 for this as reminders.  For the moment, I want to keep being able to use the pip installable version of PlasmaPy since that makes development & testing less difficult.